packages/hardhat/contracts/Ethernauts.sol

```diff
@@ -388,10 +388,6 @@ contract Ethernauts is ERC721Enumerable, Ownable, ReentrancyGuard {
         _safeMint(to, tokenId);
     }
 
```

Comment:
The following function seems to be a remnant of when we had a total supply check there. This is now done at higher levels, ensuring that this total supply error would never happen at this level.